arch/lkl/kernel/threads.c

```diff
@@ -5,7 +5,7 @@
 #include <asm/cpu.h>
 #include <asm/sched.h>
 
-static volatile int threads_counter;
```

Comment:
I have added the code changes to the latest release, so we can see if the patch works for the ARM 32bit platforms. Will let you know if it works or not.